dist/src/routes/user.route.js

```diff
@@ -67,5 +67,5 @@ usersRouter.post('/register', UserController.createUser);
 usersRouter.post('/verify', verifyToken, UserController.verifyUser);
 usersRouter.get('/users/:id', UserController.getUserById);
 usersRouter.get('/users', UserController.getAllUsers);
-usersRouter.put('/users/update//:id', UserController.updateUser);
+usersRouter.patch('/users/update/:id', verifyToken, UserController.updateUser);
```

Comment:
Your codes looks very clean and consistent. In the meantime you can add the annotation for Swagger configuration